posthog/hogql/database/schema/sessions_v2.py

```diff
@@ -57,6 +57,8 @@
     "initial_gclid": DatabaseField(name="initial_gclid"),
     "initial_fbclid": DatabaseField(name="initial_fbclid"),
     "initial_gad_source": DatabaseField(name="initial_gad_source"),
+    "initial__kx": DatabaseField(name="initial__kx"),
```

Comment:
Why 2 underscores here?

Reply:
The parameter itself has an underscore, it's `_kx`